tests/wicked/before_test.pm

```diff
@@ -13,18 +13,20 @@
 use base 'wickedbase';
 use strict;
 use testapi;
-use utils 'systemctl';
+use utils qw(zypper_call systemctl);
 use serial_terminal 'select_virtio_console';
 
 sub run {
     my ($self) = @_;
     select_virtio_console();
-    my $enable_command_logging = 'export PROMPT_COMMAND=\'logger -t openQA_CMD "$(history 1 | sed "s/^[ ]*[0-9]\+[ ]*//")"\'';
+    my $enable_command_logging
```

Comment:
same with this line, no point to change it because it is already passes Travis check